### ISSUE TITLE
Trough issue halts animal handling

### ISSUE THREAD

**Issue by lobstarooo** [rschurade/Ingnomia#145]
The save of this base (23) has one emu pasture where the trough is blocking any animal interaction, so no butchering, harvest animals (eggs) and the excess animals aren't put in a secondary pasture. Sometimes gnomes pile up trying to fill the trough. It shows green when trying to deconstruct it but it doesn't "accept" the order. I've deleted and recreated the pasture in the meantime and also tried adding a secondary pasture but the gnomes don't make use of it. For reproduction purposes I added another save (20) where the trough is still connected to the pasture, but has the same issue.

If any earlier saves are handy let me know, I can uploaded them. And if any additional info is required I can also provide that of course.

[20.zip](https://github.com/rschurade/Ingnomia/files/5788637/20.zip)
[23.zip](https://github.com/rschurade/Ingnomia/files/5788638/23.zip)

**Comment by PoweredByMave** [comment]
Confirmed this with yaks as well

Mave

**Comment by PoweredByMave** [comment]
Did you use troughs spawned in at the beginning of the game? I crafted and placed mine around the outside of the pasture and haven't ran in to issues so far, the yaks will breed as expected and be moved to a new pasture.

Mave

**Comment by lobstarooo** [comment]
The troughs were constructed later on since I started with the Gnomoria preset (only added the 2 emus to that). But I did initially place the pastures on top and only moved them down into the "pit" (also deconstructed & placed the troughs) later on, and the pastures & troughs worked better before that time. During the move my gnomes happily butchered my other yaks so that's why there's only one male left. 

I loaded a save before the move and the emu pasture works properly but the yak one already doesn't recognize at least one yak outside of the pasture, even after having 2 gnomes set only on animal husbandry and butchery. This also happens with some newly born emus. I'll attach this save too, just in case: [7.zip](https://github.com/rschurade/Ingnomia/files/5790651/7.zip)
